src/test/java/com/Project/accountManager/services/AccountServiceTest.java

```diff
@@ -0,0 +1,152 @@
+package com.Project.accountManager.services;
+
+import com.Project.accountManager.entities.Account;
+import com.Project.accountManager.entities.User;
+import com.Project.accountManager.repository.AccountRepository;
+import com.Project.accountManager.request.AccountCreateRequest;
+import com.Project.accountManager.request.AccountDepositRequest;
+import com.Project.accountManager.request.AccountWithdrawalRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class AccountServiceTest {
+
+    @Mock
+    private AccountRepository accountRepository;
+
+    @Mock
+    private UserService userService;
+
+    @InjectMocks
+    private AccountService accountService;
+
+    @BeforeEach
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testSaveOneAccount() {
+
+        Long userId = 1L;
+        AccountCreateRequest accountCreateRequest = new AccountCreateRequest();
+        accountCreateRequest.setUserId(userId);
+        accountCreateRequest.setId(1L);
+        accountCreateRequest.setAccountNumber(1324);
+        accountCreateRequest.setMoney(50);
+
+        User user = new User();
+        when(userService.getOneUser(userId)).thenReturn(user);
+
+        Account accountToSave = new Account();
+        accountToSave.setId(accountCreateRequest.getId());
+        accountToSave.setAccountNumber(accountCreateRequest.getAccountNumber());
+        accountToSave.setMoney(accountCreateRequest.getMoney());
+        accountToSave.setUser(user);
+        when(accountRepository.save(any(Account.class))).thenReturn(accountToSave);
+
+        Account savedAccount = accountService.saveOneAccount(accountCreateRequest);
```

Comment:
`saveOneAccount` , is not best practice in my opinion
maybe `createAccount` enough?